tests/gis_tests/geoapp/tests.py

```diff
@@ -157,17 +155,17 @@ def test_geometryfield(self):
 
     @skipUnlessDBFeature("supports_transform")
     def test_inherited_geofields(self):
-        "Test GeoQuerySet methods on inherited Geometry fields."
+        "Database functions on inherited Geometry fields."
         # Creating a Pennsylvanian city.
         PennsylvaniaCity.objects.create(name='Mansfield', county='Tioga', point='POINT(-77.071445 41.823881)')
 
         # All transformation SQL will need to be performed on the
         # _parent_ table.
-        qs = PennsylvaniaCity.objects.transform(32128)
+        qs = PennsylvaniaCity.objects.annotate(new_point=functions.Transform('point', srid=32128))
 
         self.assertEqual(1, qs.count())
```

Comment:
This fails on Oracle with `ORA-22901: cannot compare VARRAY or LOB attributes of an object type`.